docs/search/queries/search-functions/in_range.md

```diff
@@ -6,37 +6,42 @@ Match documents where the attribute at `path` is greater than (or equal to) `low
 
 `IN_RANGE(path, low, high, includeLow, includeHigh)`
 
-_low_ and _high_ can be numbers or strings (technically also `null`, `true` and `false`), but the data type must be the same for both.
+The values `low` and `high` can be numbers or strings, but each value must have the same data type.
 
-:::warning
-The alphabetical order of characters is not taken into account by Search, i.e. range queries in SEARCH operations against views will not follow the language rules as per the defined analyzer locale.
-:::
+- `path` (attribute path expression): The path to the attribute in the document.
```

Comment:
Consider turning some of these lists into tables. It might be easier for users to scan.